images/bootstrap/Dockerfile

```diff
@@ -73,23 +105,28 @@ RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.ta
     gcloud info | tee /workspace/gcloud-info.txt
 
 #
-# BEGIN: DOCKER IN DOCKER SETUP
+# BEGIN: PODMAN IN CONTAINER SETUP
 #
-# Install packages
 
-RUN dnf install -y \
-        kmod \
-        procps-ng \
-        moby-engine && \
-    dnf -y clean all
+RUN dnf -y update; dnf -y reinstall shadow-utils && \
+    dnf -y install crun podman --exclude container-selinux && \
+    rm -rf /var/cache /var/log/dnf* /var/log/yum.* && \
```

Comment:
why is this used instead of the higher-level `dnf -y clean all` ?

Reply:
yes makes more sense to use `dnf -y clean all` - updated